config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/freeipa/freeipa-operator
-  newTag: dev-969de0127df6c7839d172466c6a0a3bd2fcc1109
+  newName: quay.io/avisied0/freeipa-operator
```

Comment:
Is this change of image intentional?

Reply:
It is not intentional! Thank you for this, I didn't realize; but the previous value is not valid either; Until a version is generated, I am going to put here the nightly version. If I don't have a ticket related with this, I should create it.

Makefile update this value automatically when the controller container is built to match with the just built image, so that when the controller is deployed, the manifests are referencing last built image.